plugins/Goals/Menu.php

```diff
@@ -24,15 +24,13 @@ public function configureReportingMenu(MenuReporting $menu)
         $goals  = API::getInstance()->getGoals($idSite);
         $mainGoalMenu = 'Goals_Goals';
 
-        $linkToAddNewGoal = $this->urlForAction('addNewGoal', array(
-            'idGoal' => null
-        ));
+        $linkToManageGoals = $this->urlForAction('editGoals');
 
         $order = 1;
 
         if (count($goals) == 0) {
 
-            $menu->addItem($mainGoalMenu, '', $linkToAddNewGoal, 25);
+            $menu->addItem($mainGoalMenu, '', $linkToManageGoals, 25);
```

Comment:
Shouldn't we already land on the "manage goals" page anyway if there are no goals? (the manage goals page already has a "add a new goal button, having another way to do it might be confusing?)